### DISCUSSION TITLE
How can I use MMKV in openharmony multi-thread

### DISCUSSION THREAD

**Discussion by Guyuze** [Tencent/MMKV#1413] [Q&A]
i try to use it in child thread and main thread.but it doesn't work.child thread can't read data from main thread write

**Comment by lingol** [comment]
Yes, you can.

**Reply by lingol** [reply]
MMKV on OHOS is known for a dead-lock bug under v2.0.0, you can try upgrade to v2.0.1. https://github.com/Tencent/MMKV/releases/tag/v2.0.1
